index.html

```diff
@@ -28,56 +68,57 @@ <h1 class="ui inverted header">
         {% endif %}
       {% endfor %}
     </p>
-		<a class="ui big orange basic button" href="#type">Découvrez comment&nbsp;&nbsp;⬇︎</a>
```

Comment:
Remember to remove Semantic UI component bundles when the last component of that type is removed.